### PR TITLE
more blueprint customization

### PR DESCRIPTION
**Pull request by haitch**
## Description
small tweaks

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Blueprint/Management.Blueprint/Customizations/BlueprintManagementClient.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        /// </param>
+        /// <param name='handlers'>
+        /// Optional. The delegating handlers to add to the http client pipeline.
+        /// </param>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        public BlueprintManagementClient(
+            Uri baseUri, ServiceClientCredentials credentials, string apiVersionOverride, params DelegatingHandler[] handlers) : this(baseUri, credentials, handlers)
+        {
```
Comment (dsgouda) [inline src/SDKs/Blueprint/Management.Blueprint/Customizations/BlueprintManagementClient.cs]: @haitch is the nuget package already published?

Comment (haitch) [reply src/SDKs/Blueprint/Management.Blueprint/Customizations/BlueprintManagementClient.cs]: yes, we did publish yesterday. this just add one more constructor, not impacting existing constructors.

Comment (dsgouda) [reply src/SDKs/Blueprint/Management.Blueprint/Customizations/BlueprintManagementClient.cs]: In that case, version number must be updated in csproj and AssemblyInfo.cs files

File: src/SDKs/Blueprint/Management.Blueprint/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -11,8 +11,8 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Microsoft")]
 [assembly: AssemblyProduct("Azure .NET SDK")]
-[assembly: AssemblyVersion("0.9.0.0")]
-[assembly: AssemblyFileVersion("0.9.0.0")]
+[assembly: AssemblyVersion("0.9.1.0")]
```
Comment (dsgouda) [inline src/SDKs/Blueprint/Management.Blueprint/Properties/AssemblyInfo.cs]: Please leave the `AssemblyVersion` unchanged, `AssemblyFileVersion` should be bumped

File: src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj [hunk 3]
```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Blueprint</PackageId>
     <Description>Provides Microsoft Azure Blueprint management operations including the ability to create, update and delete Blueprint definition and assignment.</Description>
-    <Version>0.9.0-Preview</Version>
+    <Version>0.9.1-Preview</Version>
     <!-- Remove the Preview description in PackageReleaseNotes once the preview is removed.-->
```
Comment (dsgouda) [inline src/SDKs/Blueprint/Management.Blueprint/Microsoft.Azure.Management.Blueprint.csproj]: Please update the `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Pull request merged**